crew

```diff
@@ -444,21 +444,18 @@ def unpack (meta)
   extract_dir = "#{meta[:filename]}.dir"
   target_dir = nil
   Dir.chdir CREW_BREW_DIR do
-    puts "Unpacking archive, this may take awhile..."
     Dir.mkdir("#{extract_dir}") unless Dir.exist?("#{extract_dir}")
-    case File.extname meta[:filename]
-    when '.zip'
-      if @opt_verbose then
-        system "unzip", "-v", "-d", "#{extract_dir}", meta[:filename]
-      else
-        system "unzip", "-qq", "-d", "#{extract_dir}", meta[:filename]
-      end
+    case File.basename meta[:filename]
+    when /\.zip$/i
+      puts "Unpacking archive using zip, this may take awhile..."
+      _verbopt = @opt_verbose ? 'v' : 'qq'
+      system "unzip", _verbopt, "-d", "#{extract_dir}", meta[:filename]
+    when /\.(tar|tbz|tgz)/i
+      puts "Unpacking archive using tar, this may take awhile..."
+      _verbopt = @opt_verbose ? 'v' : ''
+      system "tar", "x#{_verbopt}f", meta[:filename], "-C", "#{extract_dir}"
     else
-      if @opt_verbose then
-        system "tar", "xvf", meta[:filename], "-C", "#{extract_dir}"
-      else
-        system "tar", "xf", meta[:filename], "-C", "#{extract_dir}"
-      end
+      puts "Not unpacking #{File.basename meta[:filename]}"
```

Comment:
No need to output anything here.  Keep in mind whenever you make changes to the `crew` command, you will also need to bump the version in `lib/const.rb`.

Reply:
Right, I will remove it: I was not sure whatever you guys require `else` clauses in switch statements. I'll bump the version by 0.01, i suppose?